cvat-ui/src/cvat-core-wrapper.ts

```diff
@@ -8,6 +8,8 @@ const cvat: any = _cvat;
 
 cvat.config.backendAPI = '/api/v1';
 cvat.config.origin = window.location.origin;
+cvat.config.uploadChunkSize = 100;
+(window as any || global as any).cvat = cvat;
```

Comment:
nodejs throws error this line:

```
> (window || global)
Uncaught ReferenceError: window is not defined
```

Let's keep only ``window`` for now.

Reply:
Or maybe to write ``this.cvat = cvat``. Works in both nodejs and browser

Reply:
I've tried to use `this.cvat = cvat` but got `can not set property cvat of undefined` error in browser using docker container. Maybe we can use `globalThis.cvat = cvat`, it works both on nodejs and browser.